src/test/java/it/pagopa/ecommerce/commons/TransactionTestUtils.java

```diff
@@ -35,9 +37,14 @@ public class TransactionTestUtils {
     public static TransactionActivationRequested transactionActivationRequested(String creationDate) {
         return new TransactionActivationRequested(
                 new TransactionId(UUID.fromString(TRANSACTION_ID)),
-                new RptId(RPT_ID),
-                new TransactionDescription(DESCRIPTION),
-                new TransactionAmount(AMOUNT),
+                Arrays.asList(
+                        new NoticeCode(
+                                null,
```

Comment:
Shouldn't it be
```suggestion
                                new PaymentToken(null),
```
?

Reply:
You're right